src/sync_widgets.rs

```diff
@@ -4,46 +4,53 @@
 
 extern crate gtk;
 
-use gtk::prelude::*;
-
-fn main() {
-    if gtk::init().is_err() {
-        println!("Failed to initialize GTK.");
-        return;
+#[cfg(feature = "gtk_3_10")]
```

Comment:
If the only reason for this is `Builder::new_from_string` we should rather bind the `add_from_string` method, which does not require 3.10.